### PR TITLE
update lightning conflict detection recording view

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)
Update lightning conflict detection recording view.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/17532
- Other reference link(s): https://github.com/pingcap/tidb/pull/53289

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lyzx2001 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -146,15 +145,13 @@ strategy = ""
...
-#  - none: 不检测重复记录。该模式是两种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
-#  - remove：如果写入的数据 A 和 B 存在 Primary Key 或 Unique Key 冲突，
-#            则会将 A 和 B 这两条冲突数据从目标表移除，同时记录到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
-#            你可以根据业务需求选择正确的记录重新手动写入到目标表中。注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。
-#            如果版本过低，则会启用 'none' 模式。
-# 默认值为 'none'。
+#  - none：不检测重复记录。
+#          如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
+#          如果 `duplicate-resolution` 设置为 `none` 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。
+#  - remove：如果 `duplicate-resolution` 设置为 `remove` 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 "replace" 开启新版冲突检测。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-configuration.md:154]:
```suggestion
#  - 'none'：不检测重复记录。
#          如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
#          如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。
#  - 'remove'：如果 `duplicate-resolution` 设置为 'remove' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 "replace" 开启新版冲突检测。
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 2]
```diff
@@ -50,7 +49,7 @@ strategy = ""
 backend = "local"
 
 # 冲突数据处理方式
-# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
+# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。如果 `duplicate-resolution` 设置为 none 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。详情参考 <https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:52]:
```suggestion
# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。详情参考 <https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```

File: tidb-lightning/tidb-lightning-error-resolution.md [hunk 3]
```diff
@@ -105,26 +105,38 @@ CREATE TABLE conflict_error_v1 (
...
-`type_error_v1` 记录由 `lightning.max-error` 配置项管理的所有[类型错误 (Type error)](#类型错误-type-error)。每个错误一行。
+`type_error_v1` 表记录由 `lightning.max-error` 配置项管理的所有[类型错误 (Type error)](#类型错误-type-error)。每个错误一行。
+
+`conflict_error_v3` 表记录物理导入模式的 `conflict` 配置组的后置检测冲突错误，每对冲突占两行。
 
-`conflict_error_v2` 记录物理导入模式的 `conflict` 配置组的冲突错误，每对冲突占两行。
+`conflict_records` 表记录逻辑导入模式和物理导入模式 `conflict` 配置组的前置检测冲突错误，每个错误占一行。
 
-`conflict_records` 记录逻辑导入模式和物理导入模式 `conflict` 配置组的冲突错误，每个错误占一行。
+`conflict_view` 视图记录物理导入模式和逻辑导入模式 `conflict` 配置组的前置和后置检测冲突错误，是 `conflict_error_v3` 表和 `conflict_records` 表的 `UNION`。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-error-resolution.md:139]:
```suggestion
`conflict_view` 视图记录物理导入模式和逻辑导入模式 `conflict` 配置组的前置和后置检测冲突错误，是通过对 `conflict_error_v3` 表和 `conflict_records` 表进行 `UNION` 操作生成的。
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 4]
```diff
@@ -50,7 +49,7 @@ strategy = ""
 backend = "local"
 
 # 冲突数据处理方式
-# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
+# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。详情参考 <https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
 duplicate-resolution = 'remove'
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:53]:
```suggestion
duplicate-resolution = 'none'
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -146,15 +145,13 @@ strategy = ""
...
-#  - remove：如果写入的数据 A 和 B 存在 Primary Key 或 Unique Key 冲突，
-#            则会将 A 和 B 这两条冲突数据从目标表移除，同时记录到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
-#            你可以根据业务需求选择正确的记录重新手动写入到目标表中。注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。
-#            如果版本过低，则会启用 'none' 模式。
-# 默认值为 'none'。
+#  - 'none'：不检测重复记录。
+#          如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
+#          如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。
+#  - 'remove'：如果 `duplicate-resolution` 设置为 'remove' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 "replace" 开启新版冲突检测。
 # duplicate-resolution = 'none'
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-configuration.md:155]:
```suggestion
# 默认值为 'none'。
# duplicate-resolution = 'none'
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 6]
```diff
@@ -50,8 +49,8 @@ strategy = ""
 backend = "local"
 
 # 冲突数据处理方式
-# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
-duplicate-resolution = 'remove'
+# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。详情参考 <https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```
Comment (lyzx2001) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:52]:
```suggestion
# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。如果 `duplicate-resolution` 设置为 `remove` 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 "replace" 开启新版冲突检测。详情参考 <https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 7]
```diff
@@ -50,8 +49,8 @@ strategy = ""
 backend = "local"
 
 # 冲突数据处理方式
-# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
-duplicate-resolution = 'remove'
+# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。如果 `duplicate-resolution` 设置为 `remove` 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 "replace" 开启新版冲突检测。详情参考 <https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:52]:
```suggestion
# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。如果 `duplicate-resolution` 设置为 'remove' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 "replace" 开启新版冲突检测。详情参考 <https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 8]
```diff
@@ -50,8 +49,8 @@ strategy = ""
 backend = "local"
 
 # 冲突数据处理方式
-# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
-duplicate-resolution = 'remove'
+# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。如果 `duplicate-resolution` 设置为 'remove' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 "replace" 开启新版冲突检测。详情参考 <https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:52]:
```suggestion
# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。
# 如果 `duplicate-resolution` 设置为 'none' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 ""。
# 如果 `duplicate-resolution` 设置为 'remove' 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 "replace" 开启新版冲突检测。
# 详情参考 <https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```

File: tidb-lightning/tidb-lightning-error-resolution.md [hunk 9]
```diff
@@ -105,26 +105,38 @@ CREATE TABLE conflict_error_v1 (
...
-`type_error_v1` 记录由 `lightning.max-error` 配置项管理的所有[类型错误 (Type error)](#类型错误-type-error)。每个错误一行。
+`type_error_v1` 表记录由 `lightning.max-error` 配置项管理的所有[类型错误 (Type error)](#类型错误-type-error)。每个错误一行。
+
+`conflict_error_v3` 表记录物理导入模式的 `conflict` 配置组的后置检测冲突错误，每对冲突占两行。
 
-`conflict_error_v2` 记录物理导入模式的 `conflict` 配置组的冲突错误，每对冲突占两行。
+`conflict_records` 表记录逻辑导入模式和物理导入模式 `conflict` 配置组的前置检测冲突错误，每个错误占一行。
 
-`conflict_records` 记录逻辑导入模式和物理导入模式 `conflict` 配置组的冲突错误，每个错误占一行。
+`conflict_view` 视图记录物理导入模式和逻辑导入模式 `conflict` 配置组的前置和后置检测冲突错误，是通过对 `conflict_error_v3` 表和 `conflict_records` 表进行 `UNION` 操作生成的。
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-error-resolution.md:140]: `conflict_error_v1` 是否需要修改为 `conflict_error_v3`：
https://github.com/pingcap/docs-cn/blob/2ac3dcf575f7501c901e80a5bcec9156550bf221/tidb-lightning/tidb-lightning-error-resolution.md?plain=1#L283-L286

https://github.com/pingcap/docs-cn/blob/2ac3dcf575f7501c901e80a5bcec9156550bf221/tidb-lightning/tidb-lightning-physical-import-mode-usage.md?plain=1#L162-L165

Comment (qiancai) [reply tidb-lightning/tidb-lightning-error-resolution.md:140]: @lyzx2001 PTAL

Comment (lyzx2001) [reply tidb-lightning/tidb-lightning-error-resolution.md:140]: modified

**Review by Oreoxmt — APPROVED** [review]

File: dm/task-configuration-file-full.md [hunk 10]
```diff
@@ -137,8 +137,8 @@ loaders:                             # load 处理单元的运行配置参数
     # 表示遇到冲突数据时不进行处理。该模式性能最佳，但下游数据库会遇到数据索引不一致的问题。
     # - "manual"。对应 TiDB Lightning 物理导入模式冲突数据检测的 "remove" 选项
```
Comment (qiancai) [inline dm/task-configuration-file-full.md:138]:
```suggestion
    # - "manual"。对应 TiDB Lightning 物理导入模式冲突数据检测的 "replace" 选项 
```

**Pull request merged**